dipy/align/streamwarp.py

```diff
@@ -188,6 +188,8 @@ def bundlewarp(
         ty = ty.astype(float)
         deformed_bundle.append(ty)
         warp.append(pr)
+        
+    warp = pd.DataFrame(warp, columns=["gaussian_kernel", "transforms"])
```

Comment:
if pandas is not installed, the function will fail. you might need to tests this